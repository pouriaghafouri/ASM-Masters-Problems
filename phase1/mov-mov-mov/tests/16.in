5869903239624274225 -3191544564728186584
