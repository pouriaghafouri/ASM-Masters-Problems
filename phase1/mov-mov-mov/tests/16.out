9061447804352460809
